public_src/components/toolbar/toolbar.component.ts

```diff
@@ -170,6 +170,21 @@ export class ToolbarComponent {
     window.open(josmHref, '_blank');
   }
 
+ 
+  private openInIdEditor(selection: object): void {
+    var zoomlevel=this.mapSrv.map.getZoom();
+    if(zoomlevel<=12)
+    {
+     zoomlevel=13;
+    }
+    const idHref = 
+    'https://www.openstreetmap.org/edit?editor=id#map='+
+    zoomlevel+'/'+
+    selection['lat']+'/'+
+    selection['lon'];   
+    window.open(idHref);
+  }
```

Comment:
Please enable linter and use consistent style (bracket style, whitespace)...
![image](https://user-images.githubusercontent.com/6165660/37019984-91d7f1b6-211a-11e8-9751-05e8184fa692.png)